app/models/task.py

```diff
@@ -1,5 +1,33 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from datetime import datetime
 from ..db import db
+from sqlalchemy import ForeignKey
+from typing import Optional
+from app.models.goal import Goal
+from app.routes.route_utilities import check_complete
 
 class Task(db.Model):
+    
+    __tablename__ = 'tasks'
```

Comment:
It doesn't seem as though you are using tablename anywhere else so we don't need to include this!